cmd/podman/remoteclientconfig/config_linux.go

```diff
@@ -0,0 +1,8 @@
+package remoteclientconfig
+
+import (
+	"os"
+	"path/filepath"
+)
+
+var configFilePath = filepath.Join(os.Getenv("HOME"), ".config", "containers", remoteConfigFileName)
```

Comment:
HOME is not portable, even across "unix'es"

`os.LookupEnv(XDG_CONFIG_HOME)` should be first check.
If needed, then
```
usr, _ := user.Current()
dir := usr.HomeDir
```

Reply:
Remember this has to work on Windows also.

Reply:
it seems os.UserHomeDir is preferred now -> https://golang.org/pkg/os/#UserHomeDir

it should also be noted that this is for unix only ... windows I believe will be different and I frankly don't know where it should go in that case.

Reply:
According to that link Windows would do the right thing as well.
```
On Unix, including macOS, it returns the $HOME environment variable. On Windows, it returns %USERPROFILE%. On Plan 9, it returns the $home environment variable. 
```

Reply:
yeah but the more interesting question is where in window's dir structure should it go?